.changeset/green-cherries-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@smartcontractkit/operator-ui': minor
```

Comment:
Unsure if this is the correct kind of semver bump, but `patch` didn't seem right since this isn't really a bug fix.

Reply:
Whats the reasoning behind the change? If its removal of a deprecated property then its a major change

Reply:
Yup it's a removal of the `maxGasPriceGWei` field on the VRF job spec. If that's major I can change the changeset to reflect that.

Reply:
Oh, does the UI actually interact with that field directly? It is not just an implementation detail inside of a TOML string?

Reply:
> Oh, does the UI actually interact with that field directly? It is not just an implementation detail inside of a TOML string?

The UI does a GQL query and specifies each field in the job spec separately. Since the field no longer exists in CL core (and the resolvers have been updated) this PR removes the `maxGasPriceGWei` from the GQL query. Otherwise I believe it'll error out.

Reply:
Is there a more-compatible middle ground here where we don't error but also don't lie about the field value?

Reply:
I don't know, but if it is a breaking change then it is MAJOR by semver definition, no? So we should either mark it as such, or try to make a minor, non-breaking compromise that deprecates it as permanently null or something :shrug:

Reply:
Also, the UI is not the only consumer of this API AFAIK, and since it iss open we can't really know who is doing what with it.

Reply:
I think marking as "major" makes sense, can do that.

Reply:
> Also, the UI is not the only consumer of this API AFAIK, and since it is open we can't really know who is doing what with it.

While this is true, isn't this changeset for the UI? Or is this changeset supposed to be for the CL GQL API? My understanding is that it was the former. The UI doesn't really hold any GQL schemas, so I'm not super sure what it means for a change to be breaking vs. non-breaking for the UI. Maybe I'm misunderstanding something though.

Reply:
True, I was thinking to broadly. I'm not sure how this is supposed to work. cc @HenryNguyen5

Reply:
Its just for the UI, if we’re removing functionality that used to exist its definitely a breaking change

Reply:
After a convo w/ Henry, going to bump down the change to 'minor' because setting it to 'major' will bring it to 1.0, and doesn't look like we want to do that yet.